### DISCUSSION TITLE
Consulta TP10 complejidad funciones

### DISCUSSION THREAD

**Discussion by martinjguevara** [INGCOM-UNRN-P1/cursada-2023#217] [Preguntas y respuestas] [answered]
Cuando tengo el uso de una función (Largo seguro) dentro de una función (Inserción cadena), para calcular la complejidad computacional, tengo que calcular la complejidad de la función largo seguro y agregarla a la de inserción? o como sería? Ademas tengo que hacer todo el análisis de la complejidad temporal y espacial de Largo seguro en el mismo archivo?

**Comment by martinvilu** [comment, marked as answer]
> Cuando tengo el uso de una función (Largo seguro) dentro de una función (Inserción cadena), para calcular la complejidad computacional, tengo que calcular la complejidad de la función largo seguro y agregarla a la de inserción?

Sí

> Ademas tengo que hacer todo el análisis de la complejidad temporal y espacial de Largo seguro en el mismo archivo?

Sí

Le tenés que dar algún valor de costo, y siendo que es código propio. Es necesario analizarlo.
